lib/inspec/profile.rb

```diff
@@ -124,6 +127,10 @@ def version
       metadata.params[:version]
     end
 
+    def cache_resources?
+      metadata.params[:cache_resources] || false
```

Comment:
`||` logic with default boolean values always makes me nervous. If someone decides to make this default behavior and just changes the `false` to `true`, it will never be able to be set to false:

```
[1] pry(main)> a = true
=> true
[2] pry(main)> a || true
=> true
[3] pry(main)> a = false
=> false
[4] pry(main)> a || true
=> true
```

To avoid a silly regression like that, how about we change this to use `#fetch` instead?

```ruby
metadata.params.fetch(:cache_resources, false)
```